pallets/funding/src/functions/6_settlement.rs

```diff
@@ -448,8 +448,11 @@ impl<T: Config> Pallet<T> {
 		vesting_time: BlockNumberFor<T>,
 	) -> DispatchResult {
 		UserMigrations::<T>::try_mutate(project_id, origin, |maybe_migrations| -> DispatchResult {
-			let migration_origin =
-				MigrationOrigin { user: T::AccountId32Conversion::convert(origin.clone()), id, participation_type };
+			let multilocation_user = MultiLocation::new(
+				0,
+				X1(AccountId32 { network: None, id: T::AccountId32Conversion::convert(origin.clone()) }),
+			);
```

Comment:
Could choose to create the ML via tuple().into() implementation (see [example](https://wiki.polkadot.network/docs/learn/xcm/fundamentals/multilocation-example))